docker-compose.yml

```diff
@@ -221,10 +226,31 @@ services:
         command: ["--config=/conf/collector-config.yaml", "--log-level=DEBUG"]
         networks:
             - demo
+        # depends_on:
+        #     - satellite
         volumes:
             - ./collector/collector-config.yaml:/conf/collector-config.yaml
         ports:
-            - "8889:8889"   # Prometheus exporter metrics
-            - "55678:55678" # OpenCensus receiver
+            - "0.0.0.0:8889:8889"   # Prometheus exporter metrics
+            - "0.0.0.0:55678:55678" # OpenCensus receiver
+            - "0.0.0.0:55680:55680" # OTLP receiver
+    # optionally configure a satellite here
+    # until OTLP ingest is shipped, this requires building a Docker container
+    # locally. This can be achieved via the following:
+    # git clone git@github.com:lightstep/lightstep && cd lightstep/go/src/github.com/lightstep/collector
+    # docker build -t satlocal:latest .
```

Comment:
I had to run this from `lightstep/go/src/github.com/lightstep` and specify the Docker file with -f. E.g. `docker build -f collector/Dockerfile -t satlocal:latest .`.